unittests/rpc/handlers/NFTsByIssuerTest.cpp

```diff
@@ -0,0 +1,359 @@
+//------------------------------------------------------------------------------
+/*
```

Comment:
@godexsoft generally these tests need help. could use your help on this too

Reply:
Yep, can go thru them together on the call.